### PR TITLE
Ammo box loot action

### PR DESCRIPTION
**Pull request by jasonmoonen**
^^ title

### CODE REVIEW

**Comment by QuentinGruber** [comment]
Hey @jasonmoonen thx for your contribution, we have a release tonight after that we could review your PR and merge it :cowboy_hat_face:

File: src/servers/ZoneServer2016/zonepackethandlers.ts [hunk 1]
```diff
@@ -2256,6 +2256,10 @@ export class ZonePacketHandlers {
           "Parts may be required. Open vehicle loadout."
         );
         break;
+      case ItemUseOptions.UNPACK:
+        server.useAmmoBox(client, item);
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zonepackethandlers.ts:2260]: If there were one in the original game then yes

File: src/servers/ZoneServer2016/zoneserver.ts [hunk 2]
```diff
@@ -6106,6 +6106,22 @@ export class ZoneServer2016 extends EventEmitter {
     }, 600000);
   }
 
+  useAmmoBox(client: Client, item: BaseItem) {
+    if (!this.removeInventoryItem(client.character, item)) return;
+
+    // Configure amount of ammo? Should be 60 by default.
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zoneserver.ts:6112]: Yep should be in the config file

Comment (Z1Meme) [reply src/servers/ZoneServer2016/zoneserver.ts:6110]: Ammo boxes weren't used in this version and were leftover from 2015 / maybe early 2016, so if these are used it'll proabably only be on modded servers so idk if we should put it in config or not. Pistol bullets have the same model as the ammo box so it'd be confusing to spawn them with eachother.

File: src/servers/ZoneServer2016/zonepackethandlers.ts (hunk 1 above)
Comment (Z1Meme) [reply src/servers/ZoneServer2016/zonepackethandlers.ts:2260]: There was

**Review by Z1Meme — APPROVED** [review]

**Comment by Z1Meme** [comment]
@jasonmoonen Merging now, thanks!

**Pull request merged**